### ISSUE TITLE
SplitButton: aria-expanded is not correct

### ISSUE THREAD

**Issue by tri-harmoney** [primefaces/primeng#15752]
### Describe the bug

If the TieredMenu is opened `aria-expanded` is set to true, but when the TieredMenu is closed using the escape key or by clicking outside the element it is not set to false.

### Environment

macOS 14.5 (23F79)

### Angular version

17.3.7

### PrimeNG version

17.16.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18.13.0

### Steps to reproduce the behavior

1. Open the TieredMenu by clicking on the chevron.
2. Check `aria-expanded` is `true`
3. Close TieredMenu by pressing escape button or by clicking outside element
4. 'aria-expanded` is not set to `false`

### Expected behavior

'aria-expanded` should be `false` when menu is closed.
